moped-database/migrations/1662751229997_create_table_public_moped_tags/up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE "public"."moped_tags" (
+	"id" serial, 
+	"name" text NOT NULL, 
+	"type" text NOT NULL, 
+	"slug" text NOT NULL, 
+	"is_deleted" boolean NOT NULL DEFAULT false, 
+	PRIMARY KEY ("id") , 
+	UNIQUE ("name"), UNIQUE ("slug"));
```

Comment:
unique slug - thank you for enforcing that 1:1 relationship between tag and slugs for URLs or what-have-you! 🐌